include/podio/ROOTFrameReader.h

```diff
@@ -40,7 +41,7 @@ struct CollectionReadBuffers;
  * This class has the function to read available data from disk
  * and to prepare collections and buffers.
  **/
-class ROOTFrameReader {
+class ROOTFrameReader : public EDMDefinitionHolder {
```

Comment:
) I would not use mixins in this case. I do not see a reason where and why the `EDMDefinitionCollector` should be exposed as an interface. It can stay internal. For the other one. I wouldn't do it either.

Reply:
This is changed to internal members now.